public/index.html

```diff
@@ -23,17 +23,13 @@ <h5 id="campaignType">Campaign Type</h5>
 
   <div class="container text-center">
     <div class="row">
-      <div class="col-sm-4">
+      <div class="col-sm-4" style="border:1px">
```

Comment:
it's not great to mix classNames and inline styles

Maybe there is a bootstrap class you can add to get a border